Index: texlive-20091107-source/texk/texlive/linked_scripts/Makefile.in
===================================================================
--- texlive-20091107-source.orig/texk/texlive/linked_scripts/Makefile.in
+++ texlive-20091107-source/texk/texlive/linked_scripts/Makefile.in
@@ -156,83 +156,30 @@ top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 bin_SCRIPTS = 
-scriptxdir = ${prefix}/texmf/scripts
+scriptxdir = ${prefix}/share/texmf/scripts
 nobase_dist_scriptx_SCRIPTS = \
 	a2ping/a2ping.pl \
 	getnonfreefonts/getnonfreefonts.pl \
-	simpdftex/simpdftex \
 	tetex/e2pall.pl \
 	tetex/texdoctk.pl \
 	texdoc/texdoc.tlu \
-	texlive/rungs.tlu \
-	texlive/tlmgr.pl
 
-scriptdir = ${prefix}/texmf-dist/scripts
+scriptdir = ${prefix}/share/texmf-dist/scripts
 nobase_dist_script_SCRIPTS = \
-	accfonts/mkt1font \
-	accfonts/vpl2ovp \
-	accfonts/vpl2vpl \
-	bengali/ebong.py \
-	cachepic/cachepic.tlu \
-	context/lua/luatools.lua \
-	context/lua/mtxrun.lua \
-	context/ruby/texmfstart.rb \
-	context/stubs/unix/context \
-	context/stubs/unix/ctxtools \
-	context/stubs/unix/makempy \
-	context/stubs/unix/mpstools \
-	context/stubs/unix/mptopdf \
-	context/stubs/unix/mtxtools \
-	context/stubs/unix/pdftools \
-	context/stubs/unix/pstopdf \
-	context/stubs/unix/rlxtools \
-	context/stubs/unix/runtools \
-	context/stubs/unix/texexec \
-	context/stubs/unix/texfont \
-	context/stubs/unix/textools \
-	context/stubs/unix/texutil \
-	context/stubs/unix/tmftools \
-	context/stubs/unix/xmltools \
 	dviasm/dviasm.py \
-	epspdf/epspdf \
-	epspdf/epspdftk \
-	epstopdf/epstopdf.pl \
-	fig4latex/fig4latex \
 	findhyph/findhyph \
-	fontools/afm2afm \
-	fontools/autoinst \
-	fontools/cmap2enc \
-	fontools/font2afm \
-	fontools/ot2kpx \
-	fontools/pfm2kpx \
-	fontools/showglyphs \
 	fragmaster/fragmaster.pl \
-	glossaries/makeglossaries \
 	latex2man/latex2man \
-	latexmk/latexmk.pl \
 	listings-ext/listings-ext.sh \
 	mkjobtexmf/mkjobtexmf.pl \
-	mkgrkindex/mkgrkindex \
-	oberdiek/pdfatfi.pl \
-	pax/pdfannotextractor.pl \
 	pdfcrop/pdfcrop.pl \
-	perltex/perltex \
 	pkfix/pkfix.pl \
 	pkfix-helper/pkfix-helper \
-	ppower4/pdfthumb.tlu \
-	ppower4/ppower4.tlu \
-	pst-pdf/ps4pdf \
-	pst2pdf/pst2pdf.pl \
 	purifyeps/purifyeps \
-	splitindex/perl/splitindex.pl \
-	svn-multi/svn-multi.pl \
 	texcount/texcount.pl \
 	texdiff/texdiff \
 	texdirflatten/texdirflatten \
 	texloganalyser/texloganalyser \
-	thumbpdf/thumbpdf.pl \
-	ulqda/ulqda.pl \
-	vpe/vpe.pl
 
 all: all-am
 
@@ -542,8 +489,8 @@ uninstall-am: uninstall-binSCRIPTS unins
 
 install-data-hook:
 	case "$(bindir)" in \
-	  */bin) $(MAKE) $(AM_MAKEFLAGS) REL=.. install-links;; \
-	  */bin/*) $(MAKE) $(AM_MAKEFLAGS) REL=../.. install-links;; \
+	  */bin) $(MAKE) $(AM_MAKEFLAGS) REL=../share install-links;; \
+	  */bin/*) $(MAKE) $(AM_MAKEFLAGS) REL=../../share install-links;; \
 	  *) echo "strange directory '$(bindir)' for linked scripts" >&2; \
 	     exit 1;; \
 	esac
@@ -573,10 +520,7 @@ install-links:
 	  $(LN_S) epstopdf repstopdf && \
 	  rm -f getnonfreefonts-sys && \
 	  echo "creating link 'getnonfreefonts-sys' -> 'getnonfreefonts'" && \
-	  $(LN_S) getnonfreefonts getnonfreefonts-sys && \
-	  rm -f man && \
-	  echo "creating link 'man' -> '$(REL)/texmf/doc/man'" && \
-	  (ln -s $(REL)/texmf/doc/man man || :)
+	  $(LN_S) getnonfreefonts getnonfreefonts-sys
 
 uninstall-hook:
 	@for s in $(nobase_dist_scriptx_SCRIPTS) $(nobase_dist_script_SCRIPTS); do \
Index: texlive-20091107-source/texk/tetex/Makefile.in
===================================================================
--- texlive-20091107-source.orig/texk/tetex/Makefile.in
+++ texlive-20091107-source/texk/tetex/Makefile.in
@@ -197,7 +197,6 @@ dist_bin_SCRIPTS = \
 	kpsetool \
 	kpsewhere \
 	ps2frag \
-	ps4pdf \
 	pslatex \
 	rubibtex \
 	rumakeindex \
