Index: texlive-20091107-source/texk/tetex/Makefile.in
===================================================================
--- texlive-20091107-source.orig/texk/tetex/Makefile.in
+++ texlive-20091107-source/texk/tetex/Makefile.in
@@ -253,14 +253,14 @@ dist_man5_MANS = \
 	fmtutil.cnf.5 \
 	updmap.cfg.5
 
-tetexdocdir = ${prefix}/texmf/doc/tetex
+tetexdocdir = ${prefix}/share/texmf/doc/tetex
 dist_tetexdoc_DATA = \
 	doc/TETEXDOC.dvi \
 	doc/TETEXDOC.pdf \
 	doc/TETEXDOC.ps \
 	doc/teTeX-FAQ
 
-texconfdir = ${prefix}
+texconfdir = ${prefix}/share/
 nobase_dist_texconf_DATA = \
 	texmf/texconfig/README \
 	texmf/texconfig/g/generic \
@@ -268,10 +268,10 @@ nobase_dist_texconf_DATA = \
 	texmf/texconfig/v/vt100 \
 	texmf/texconfig/x/xterm
 
-texconfigdir = ${prefix}/texmf/texconfig
+texconfigdir = ${prefix}/share/texmf/texconfig
 dist_texconfig_SCRIPTS = tcfmgr
 dist_texconfig_DATA = tcfmgr.map
-web2cdir = ${prefix}/texmf/web2c
+web2cdir = ${prefix}/share/texmf/web2c
 dist_web2c_DATA = fmtutil.cnf
 EXTRA_DIST = README.a2ping README.texdoctk doc/Makefile \
 	doc/TETEXDOC.bib doc/TETEXDOC.tex fontinst.bat pdfetex-pl.pool \
Index: texlive-20091107-source/texk/dvipsk/Makefile.in
===================================================================
--- texlive-20091107-source.orig/texk/dvipsk/Makefile.in
+++ texlive-20091107-source/texk/dvipsk/Makefile.in
@@ -376,7 +376,7 @@ dist_prologues = \
 	tex.lpro \
 	texps.lpro
 
-prologdir = ${prefix}/texmf/dvips/base
+prologdir = ${prefix}/share/texmf/dvips/base
 prologues = $(dist_prologues:.lpro=.pro) texc.pro
 SUFFIXES = .pro .lpro
 EXTRA_DIST = $(dist_prologues) texc.script dvips.test pfbincl.test \
Index: texlive-20091107-source/texk/gsftopk/Makefile.in
===================================================================
--- texlive-20091107-source.orig/texk/gsftopk/Makefile.in
+++ texlive-20091107-source/texk/gsftopk/Makefile.in
@@ -256,7 +256,7 @@ gsftopk_SOURCES = \
 gsftopk_DEPENDENCIES = $(KPATHSEA_DEPEND)
 LDADD = $(KPATHSEA_LIBS)
 dist_gsftopkpsheader_DATA = render.ps
-gsftopkpsheaderdir = ${prefix}/texmf/dvips/gsftopk
+gsftopkpsheaderdir = ${prefix}/share/texmf/dvips/gsftopk
 nodist_man1_MANS = gsftopk.1
 DISTCLEANFILES = gsftopk.1 sedscript
 EXTRA_DIST = gsftopk1.sed mksedscript
Index: texlive-20091107-source/texk/bibtex8/Makefile.in
===================================================================
--- texlive-20091107-source.orig/texk/bibtex8/Makefile.in
+++ texlive-20091107-source/texk/bibtex8/Makefile.in
@@ -263,7 +263,7 @@ bibtex8_SOURCES = \
 
 bibtex8_DEPENDENCIES = $(KPATHSEA_DEPEND)
 LDADD = $(KPATHSEA_LIBS)	
-csfdir = ${prefix}/texmf-dist/bibtex/csf/base
+csfdir = ${prefix}/share/texmf-dist/bibtex/csf/base
 dist_csf_DATA = \
 	csf/88591lat.csf \
 	csf/88591sca.csf \
@@ -274,7 +274,7 @@ dist_csf_DATA = \
 	csf/cp866rus.csf \
 	csf/csfile.txt
 
-btdocdir = ${prefix}/texmf/doc/bibtex8
+btdocdir = ${prefix}/share/texmf/doc/bibtex8
 dist_btdoc_DATA = \
 	00readme.txt \
 	HISTORY \
Index: texlive-20091107-source/texk/kpathsea/Makefile.in
===================================================================
--- texlive-20091107-source.orig/texk/kpathsea/Makefile.in
+++ texlive-20091107-source/texk/kpathsea/Makefile.in
@@ -432,7 +432,7 @@ DISTCLEANFILES = paths.h stamp-paths kpa
 kpseaccess_SOURCES = access.c
 kpsereadlink_SOURCES = readlink.c
 kpsewhich_LDADD = libkpathsea.la
-web2cdir = ${prefix}/texmf/web2c
+web2cdir = ${prefix}/share/texmf/web2c
 dist_web2c_SCRIPTS = mktexdir mktexnam mktexupd
 dist_web2c_DATA = mktex.opt mktexdir.opt mktexnam.opt
 dist_noinst_SCRIPTS = mktexlsr mktexmf mktexpk mktextfm
