Index: texlive-20080816-source/libs/teckit/lib/Makefile.am
===================================================================
--- texlive-20080816-source.orig/libs/teckit/lib/Makefile.am
+++ texlive-20080816-source/libs/teckit/lib/Makefile.am
@@ -13,5 +13,8 @@ lib_LTLIBRARIES = libTECkit_Compiler.la 
 ## libTECkit_Compiler_la_LIBADD = @LDZLIB@
 ## libTECkit_la_LIBADD = @LDZLIB@
 
+libTECkit_Compiler_la_LIBADD = @LDZLIB@
+libTECkit_la_LIBADD = @LDZLIB@
+
 include ../source/Makefile.am
 
