Index: texlive-core/texk/make/man.mk
===================================================================
--- texlive-core.orig/texk/make/man.mk
+++ texlive-core/texk/make/man.mk
@@ -28,7 +28,7 @@ txt: $(manfiles:.1=.txt)
 # Makefile can change for reasons that do not affect the man pages.
 # At present, all but VERSION should be unused.
 sedscript:
-	cp /dev/null sedscript
+	echo > sedscript
 	for f in $(kpathsea_dir)/paths.h; do \
 	  sed -n -e '/^#define/s/#define[ 	][ 	]*\([A-Z_a-z][A-Z_a-z]*\)[ 	][ 	]*\(.*\)/s%@\1@%\2%/p' \
 		$$f \
