Index: source/texk/texlive/Makefile.in
===================================================================
--- source.orig/texk/texlive/Makefile.in
+++ source/texk/texlive/Makefile.in
@@ -14,7 +14,6 @@ LINKED_SCRIPTS = \
   texmf/scripts/a2ping/a2ping.pl \
   texmf/scripts/epstopdf/epstopdf.pl \
   texmf/scripts/pkfix/pkfix.pl \
-  texmf/scripts/ps2eps/ps2eps.pl \
   texmf/scripts/simpdftex/simpdftex \
   texmf/scripts/tetex/e2pall.pl \
   texmf/scripts/tetex/texdoctk.pl \
@@ -26,7 +25,6 @@ LINKED_SCRIPTS = \
   texmf-dist/scripts/dviasm/dviasm.py \
   texmf-dist/scripts/epspdf/epsfpdf \
   texmf-dist/scripts/epspdf/epsfpdftk \
-  texmf-dist/scripts/glossaries/makeglossaries \
   texmf-dist/scripts/mkjobtexmf/mkjobtexmf.pl \
   texmf-dist/scripts/oberdiek/pdfatfi.pl \
   texmf-dist/scripts/pdfcrop/pdfcrop.pl \
