diff -ur texlive-20080816-source.orig/texk/dvipdfmx/src/pngimage.c texlive-20080816-source/texk/dvipdfmx/src/pngimage.c
--- texlive-20080816-source.orig/texk/dvipdfmx/src/pngimage.c	2008-04-21 03:43:12.000000000 +0300
+++ texlive-20080816-source/texk/dvipdfmx/src/pngimage.c	2010-05-15 18:40:54.000000000 +0300
@@ -146,7 +146,7 @@
   rewind (png_file);
   if (fread (sigbytes, 1, sizeof(sigbytes), png_file) !=
       sizeof(sigbytes) ||
-      (!png_check_sig (sigbytes, sizeof(sigbytes))))
+      (png_sig_cmp (sigbytes, 0, sizeof(sigbytes))))
     return 0;
   else
     return 1;
Only in texlive-20080816-source/texk/dvipdfmx/src: pngimage.c.orig
diff -ur texlive-20080816-source.orig/texk/web2c/xetexdir/pngimage.c texlive-20080816-source/texk/web2c/xetexdir/pngimage.c
--- texlive-20080816-source.orig/texk/web2c/xetexdir/pngimage.c	2006-07-26 19:19:45.000000000 +0300
+++ texlive-20080816-source/texk/web2c/xetexdir/pngimage.c	2010-05-15 18:40:54.000000000 +0300
@@ -101,7 +101,7 @@
   rewind (png_file);
   if (fread (sigbytes, 1, sizeof(sigbytes), png_file) !=
       sizeof(sigbytes) ||
-      (!png_check_sig (sigbytes, sizeof(sigbytes))))
+      (png_sig_cmp (sigbytes, 0, sizeof(sigbytes))))
     return 0;
   else
     return 1;
diff -ur texlive-20080816-source.orig/texk/web2c/xetexdir/XeTeX_ext.c texlive-20080816-source/texk/web2c/xetexdir/XeTeX_ext.c
--- texlive-20080816-source.orig/texk/web2c/xetexdir/XeTeX_ext.c	2008-08-01 16:54:24.000000000 +0300
+++ texlive-20080816-source/texk/web2c/xetexdir/XeTeX_ext.c	2010-05-15 18:42:22.000000000 +0300
@@ -200,7 +200,7 @@
 			+ strlen(zlib_version)
 #ifdef XETEX_OTHER
 			+ strlen(PNG_LIBPNG_VER_STRING)
-			+ strlen(png_libpng_ver)
+			+ strlen(png_get_header_ver(NULL))
 			+ strlen(xpdfVersion)
 			+ 6 * 3 /* for fontconfig version #s (won't really need 3 digits per field!) */
 #endif
@@ -225,7 +225,7 @@
 		,
 		FC_VERSION / 10000, (FC_VERSION % 10000) / 100, FC_VERSION % 100,
 		fc_version / 10000, (fc_version % 10000) / 100, fc_version % 100,
-		PNG_LIBPNG_VER_STRING, png_libpng_ver,
+		PNG_LIBPNG_VER_STRING, png_get_header_ver(NULL),
 		xpdfVersion
 #endif
 		);
diff -ur texlive-20080816-source.orig/texk/xdvipdfmx/src/pngimage.c texlive-20080816-source/texk/xdvipdfmx/src/pngimage.c
--- texlive-20080816-source.orig/texk/xdvipdfmx/src/pngimage.c	2007-11-22 18:48:06.000000000 +0200
+++ texlive-20080816-source/texk/xdvipdfmx/src/pngimage.c	2010-05-15 18:40:54.000000000 +0300
@@ -146,7 +146,7 @@
   rewind (png_file);
   if (fread (sigbytes, 1, sizeof(sigbytes), png_file) !=
       sizeof(sigbytes) ||
-      (!png_check_sig (sigbytes, sizeof(sigbytes))))
+      (png_sig_cmp (sigbytes, 0, sizeof(sigbytes))))
     return 0;
   else
     return 1;
diff -ur texlive-20080816-source.orig/texk/xdvipdfmx/src/xbb.c texlive-20080816-source/texk/xdvipdfmx/src/xbb.c
--- texlive-20080816-source.orig/texk/xdvipdfmx/src/xbb.c	2008-05-25 18:00:35.000000000 +0300
+++ texlive-20080816-source/texk/xdvipdfmx/src/xbb.c	2010-05-15 18:40:54.000000000 +0300
@@ -246,7 +246,7 @@
   unsigned char sigbytes[4];
   rewind (png_file);
   if (fread(sigbytes, 1, sizeof(sigbytes), png_file) != sizeof(sigbytes) ||
-      (!png_check_sig (sigbytes, sizeof(sigbytes)))) return 0;
+      (png_sig_cmp (sigbytes, 0, sizeof(sigbytes)))) return 0;
   else return 1;
 }             
 
