Index: texlive-core/texk/web2c/Makefile.in
===================================================================
--- texlive-core.orig/texk/web2c/Makefile.in
+++ texlive-core/texk/web2c/Makefile.in
@@ -567,7 +567,7 @@ programs = bibtex ctangle ctie cweave dv
  gftype mft $(odvicopy) $(odvitype) $(otangle) patgen $(pdftosrc) \
  pktogf pktype pltotf pooltype tangle tftopl tie $(ttf2afm) \
  vftovp vptovf weave
-engines = $(tex) $(etex) $(mf) $(mfn) $(mpost) \
+engines = $(tex) $(etex) $(mf) $(mfn) $(metapost) \
           $(omega) $(aleph) \
           $(pdftex) $(pdfetex) \
           $(xetex)
